README.md

```diff
@@ -26,7 +26,7 @@ Python 3.7+.
 
 | Common modules | Community modules |
 | -------------- | ----------------- |
-| netCDF4        | pysat >= 3.0.0    |
+| netCDF4        | pysat >= 3.0.2    |
```

Comment:
Should we require 3.0.4 now?  The change mainly affects testing in the CI environment.

Reply:
Yeah, it will allow us to remove a deprecated routine.